### DISCUSSION TITLE
Tricep exercises

### DISCUSSION THREAD

**Discussion by dinakarbl** [Ruchit13/AI-Gym-Trainer#11] [Q&A] [answered]
Need an alternative exercise to Tricep extension

**Comment by rtsh13** [comment, marked as answer]
Hey, These could be of help!
Diamond Push-Ups.
Kickbacks.
Dips.
Overhead Triceps Extensions.
Rope Pushdowns.
Bar Pushdowns.
Lying Triceps Extensions.
Close Grip Bench Presses.
